_sass/base/_markdown.scss

```diff
@@ -0,0 +1,92 @@
+.markdown {
+  padding-bottom: 60px;
+  
+  // Text Blocks
+  p {
+    margin: 10px 0;
+    font-weight: 300;
+
+    @media (min-width: $tablet) {
+      margin: 20px 0;
+    }
+  }
+  
+
+  img {
+    width: 100%;
+  }
+
+  // List Items
+  li {
+    line-height: 24px;
+  }
+
+  // Lists
+  ol,
+  ul {
+    list-style: initial;
+    list-style-position: inside;
+    color: $black;
+    font-size: 15px;
+    margin: 0 20px;
+
+    @media (max-width: 768px) {
+      margin: 0;
+    }
+  }
+
+  h1 {
+    color: $secondBrandColor;
+  }
+
+  // Section Headlines 
+  h2 {
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Headlines 
+  h3 {
+    color: $black;
+    font-size: 18px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Subheadlines
+  h4 {
+    margin: 20px 25px;
+    font-size: 16px;
+    font-weight: 300;
+    color: $dark-gray;
+    
+    @media (max-width: 768px) {
+      margin: 20px 0;
+    } 
+    // Text immediately following h6 should have same level of indention  
```

Comment:
@LisaFC could you elaborate a little about how this should look? Would you prefer that the code blocks and its lead are not on the same level? In this screenshot, should the 'Simple RPC' be on the same level of indention as 'Now let’s look at how we call our service methods.' , or should the code block be indented one level further?


![screen shot 2016-07-18 at 2 28 34 pm](https://cloud.githubusercontent.com/assets/11986887/16931083/f43f48ec-4cf3-11e6-811c-abddd77cb007.png)